contracts/UmaConditionalTokensBinaryAdapter.sol

```diff
@@ -395,49 +368,28 @@ contract UmaCtfAdapter is Auth, BulletinBoard, ReentrancyGuard {
         // Ensure the price request is event based
         optimisticOracle.setEventBased(UmaConstants.YesOrNoIdentifier, requestTimestamp, ancillaryData);
 
+        // Ensure that the dispute callback flag is set
+        optimisticOracle.setCallbacks(
+            UmaConstants.YesOrNoIdentifier,
+            requestTimestamp,
+            ancillaryData,
+            false, // DO NOT set callback on priceProposed
+            true, // DO set callback on priceDisputed
+            false // DO NOT set callback on priceSettled
```

Comment:
Do we want to consider leveraging the callback on `priceSettled` to remove need to call `resolve`?